ledger/participant-integration-api/src/test/lib/scala/platform/store/dao/JdbcLedgerDaoPartiesSpec.scala

```diff
@@ -123,18 +123,47 @@ private[dao] trait JdbcLedgerDaoPartiesSpec {
     )
   }
 
+  it should "store just offset when duplicate party update encountered" in {
+    val party = Ref.Party.assertFromString(s"Alice-${UUID.randomUUID()}")
+    val aliceDetails = PartyDetails(
+      party,
+      displayName = Some("Alice Arkwright"),
+      isLocal = true,
+    )
+    val aliceAgain = PartyDetails(
+      party,
+      displayName = Some("Alice Carthwright"),
```

Comment:
Does the `displayName` matter from the test case's perspective? If not, you could simply assign `aliceAgain` to `aliceDetails`.

Reply:
It does, the test asserts that the second `storePartyEntry` call did not alter the already existing entry (see `parties should contain allOf (aliceDetails, ...)`. We could have decided to update existing entries instead.